src/daemon/common.c

```diff
@@ -267,7 +263,12 @@ int swrite(int fd, const void *buf, size_t count) {
   pfd.revents = 0;
   if (poll(&pfd, 1, 0) > 0) {
     char buffer[32];
-    if (recv(fd, buffer, sizeof(buffer), MSG_PEEK | MSG_DONTWAIT) == 0) {
+    if (recv(fd, buffer, sizeof(buffer),
+             MSG_PEEK
+#ifndef WIN32
+                 | MSG_DONTWAIT
```

Comment:
this looks a bit ugly, how about defining MSG_DONTWAIT to 0 for win32 like
```
#ifdef WIN32
#define MSG_DONTWAIT 0
#endif
```
and then keeping the recv call unchanged?

Reply:
Done.